homework/calculate/calculate.hpp

```diff
@@ -3,5 +3,21 @@
 
 std::string calculate(const std::string& command, int first, int second) {
     // TODO: Implement your solution here and return proper value
+    if (command == "add"){
+        return std::to_string(first + second);
+    }else if (command == "subtract"){
+        return std::to_string(first - second);
+        } else if (command == "multiply"){
+        	return std::to_string(first * second);
+        	}else if (command == "divide"){
+        		if (second == 0){
+       	 		return "Division by 0";
+       	 	}else{
+       	 		return std::to_string(first / second);
+       	 		}
+        	}	
+    else{	
+        return "Invalid data";
+    }
```

Comment:
Please be consistent with spaces.

Add space between operations like '+' '-' '!=' '==' '%' etc...
Add space after while, if, else, {, }, etc...

Also wrong indentation